cmd/parca-agent/main.go

```diff
@@ -98,9 +98,10 @@ type flags struct {
 	RemoteStoreInsecureSkipVerify     bool          `kong:"help='Skip TLS certificate verification.'"`
 	RemoteStoreDebugInfoUploadDisable bool          `kong:"help='Disable debuginfo collection and upload.',default='false'"`
 	RemoteStoreBatchWriteInterval     time.Duration `kong:"help='Interval between batch remote client writes. Leave this empty to use the default value of 10s.',default='10s'"`
-
 	// Debug info configuration:
 	DebugInfoDirectories []string `kong:"help='Ordered list of local directories to search for debug info files. Defaults to /usr/lib/debug.',default='/usr/lib/debug'"`
+	// These flags are considered beta quality. Use them at your own peril.
+	DwarfUnwindingPids []int `kong:"help='Unwind stack using .eh_frame information for these processes.'"`
```

Comment:
Let's `DebugPids` with a more generic name and keep it around for later use.  e.g. enabling debug logs for any profiler attached to these pids. 

There's a maintenance cost attached to the flags since they're using facing and this seems like a nice one.

Reply:
Sounds good!